modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_claim_service_base.rb

```diff
@@ -44,6 +46,33 @@ def get_error_message(error)
         end
       end
 
+      def get_error_key(error_message)
```

Comment:
In my testing, `error_message` can come back as a `String`, so I recommending adding
```
return error_message if error_message.is_a? String
```
to the top of this method

Reply:
adding that now and removing the string check from the `get_error_message` method